### PR TITLE
New rule : Prefer Orbit UI React components over Semantic UI React components.

### PR DESCRIPTION
**Pull request by alexasselin008**
added a rule preventing the import of the semantic-ui-react components that are wrapped by orbit ui

### CODE REVIEW

File: packages/rules/docs/rules/prefer-orbit-components.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Prefer orbit components over semantic-ui-react components. (prefer-orbit-components)
```
Comment (patricklafrance) [inline packages/rules/docs/rules/prefer-orbit-components.md]: Orbit avec un "o" majuscule :)

Comment (patricklafrance) [reply packages/rules/docs/rules/prefer-orbit-components.md]: Tu le fera partout

File: packages/rules/lib/rules/prefer-orbit-components.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+const BannedComponents = ["Button", "Label"]
```
Comment (patricklafrance) [inline packages/rules/lib/rules/prefer-orbit-components.js]: Input will also be available in Orbit soon? You might want to wait before merging the PR, your call

**Comment by patricklafrance** [comment]
Please fix PR title and description

File: packages/rules/lib/rules/prefer-orbit-components.js (hunk 2 above)
Comment (alexasselin008) [reply packages/rules/lib/rules/prefer-orbit-components.js]: I'll update the rule as new components are released.

**Pull request merged**